CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 
 - (Splunk) Add `apachespark` receiver ([#5318](https://github.com/signalfx/splunk-otel-collector/pull/5318))
 
+### 💡 Enhancements 💡
+
+- (Splunk) Apache Web Server Auto Discovery: set the default discovered endpoint to match the OpenTelemetry `apachereceiver` default: `http://`endpoint`/server-status?auto` ([#5353](https://github.com/signalfx/splunk-otel-collector/pull/5353))
+  If the collector is running as a process on the host OS and the Apache Web Server is in a Docker container add `--set=splunk.discovery.extensions.docker_observer.config.use_host_bindings=true` to the command-line arguments for the discovery to create the correct endpoint.
```

Comment:
These are already know facts [here](https://github.com/signalfx/splunk-otel-collector/tree/main/internal/confmapprovider/discovery). I meant to say it holds for all discovery apps/services/dbs. May be we can skip linke16

Reply:
Good point, unfortunately, `use_host_bindings` is not explicitly listed there. To be fair I didn't read that doc carefully, but, I will add a task to review and update the doc, I think that there are a few things that we should be more explicit about.